src/common/meta/src/key/schema_name.rs

```diff
@@ -58,12 +58,17 @@ impl Default for SchemaNameKey<'_> {
 pub struct SchemaNameValue {
     #[serde(default)]
     pub ttl: Option<DatabaseTimeToLive>,
+    #[serde(default)]
+    pub extra_options: BTreeMap<String, String>,
```

Comment:
`ttl` and `extra_options` are two conversely ways to organize the options: `ttl` is to make options explicit, while `extra_options` is implicit. I think it's better to stick to one, either "flatten" the `extra_options`, make every options appeared like the "ttl", or absorb the "ttl" in the `extra_options`.

Reply:
The `ttl` setting is unique: tables without a specified `ttl` will inherit this value, and changing it will impact all such tables.

The other options act as template variables and should be separated.